packages/server/modules/core/repositories/userEmails.ts

```diff
@@ -42,6 +43,14 @@ export const createUserEmailFactory =
       await checkPrimaryEmail({ db })(rest)
     }
 
+    const existingVerifiedEmail = await findEmailFactory({ db })({
+      email,
+      verified: true
```

Comment:
this should be verified false as discussed

Reply:
I removed the check completely.
Now it throws if it finds an existing email without considering if it is verified or not.